openshift/templates/api/api-deploy.yaml

```
@@ -127,20 +127,20 @@ objects:
                 - name: PORT
                   value: ${SERVICE_PORT}
                 - name: DB_SERVICE
-                  value: ${DB_SERVICE}${SUFFIX}
+                  value: ${DB_SERVICE}
```

Would you not need the `DB_SUFFIX` here if it was defined?
```suggestion
                  value: ${DB_SERVICE}${DB_SUFFIX}
```